packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -201,8 +204,20 @@ export default class SalesforceCRMService implements CRM {
       this.log.warn("salesforceCreateEvent: No contacts found for event", event.uid, contacts);
     }
 
+    const eventWhoIds = contacts.reduce((contactIds, contact) => {
+      if (contact?.id) {
```

Comment:
We're seeing a `Cannot read properties of undefined (reading 'id')` log in this method from this. I would rather still create the salesforce event with not all contacts being associated with the event than not create one at all